test/functionalTests/runner/taskExpectations.js

```diff
@@ -255,63 +247,63 @@ module.exports = {
         "MessageData",
         (telemetry) => {
             return telemetry.data.baseData.message === "test error" &&
-            telemetry.data.baseData.severityLevel === 3;
+            telemetry.data.baseData.severityLevel === "Error";
         }
     ),
     "WinstonWarn": outputContract(
         "MessageData",
         (telemetry) => {
             return telemetry.data.baseData.message === "test warn" &&
-            telemetry.data.baseData.severityLevel === 2;
+            telemetry.data.baseData.severityLevel === "Warning";
         }
     ),
     "WinstonInfo": outputContract(
         "MessageData",
         (telemetry) => {
             return telemetry.data.baseData.message === "test info" &&
-            telemetry.data.baseData.severityLevel === 1;
+            telemetry.data.baseData.severityLevel === "Information";
         }
     ),
     "WinstonVerbose": outputContract(
         "MessageData",
         (telemetry) => {
             return telemetry.data.baseData.message === "test verbose" &&
-            telemetry.data.baseData.severityLevel === 0;
+            telemetry.data.baseData.severityLevel === "Verbose";
         }
     ),
     "WinstonDebug": outputContract(
         "MessageData",
         (telemetry) => {
             return telemetry.data.baseData.message === "test debug" &&
-            telemetry.data.baseData.severityLevel === 0;
+            telemetry.data.baseData.severityLevel === "Verbose";
         }
     ),
     "WinstonSilly": outputContract(
         "MessageData",
         (telemetry) => {
             return telemetry.data.baseData.message === "test silly" &&
-            telemetry.data.baseData.severityLevel === 0;
+            telemetry.data.baseData.severityLevel === "Verbose";
         }
     ),
     "WinstonError2": outputContract(
         "MessageData",
         (telemetry) => {
             return telemetry.data.baseData.message === "test error" &&
-            telemetry.data.baseData.severityLevel === 3;
+            telemetry.data.baseData.severityLevel === "Error";
         }
     ),
     "WinstonWarn2": outputContract(
         "MessageData",
         (telemetry) => {
             return telemetry.data.baseData.message === "test warn" &&
-            telemetry.data.baseData.severityLevel === 2;
+            telemetry.data.baseData.severityLevel === "Warning";
```

Comment:
All these string might be better defined as constants somewhere, instead of repeating

Reply:
The actual source code uses an enumerator "KnownSeverityLevel", I usually prefer to hardcode the values in the tests to ensure if the enumerator ever change the tests are affected, but I'm fine either way